pycqed/measurement/calibration/two_qubit_gates.py

```diff
@@ -1399,9 +1433,7 @@ def get_meas_objs_from_task(self, task):
         :param task: a task dictionary
         :return: list of qubit objects (if available) or names
         """
-        # FIXME is this correct? it will prevent us from doing
-        #  preselection/reset on the other qubit
-        qbs = self.get_qubits([task['qbr']])
+        qbs = self.get_qubits([task['qbc'], task['qbt']])
         return qbs[0] if qbs[0] is not None else qbs[1]
```

Comment:
I thought that you had a list of `qubits_to_measure` in your dynamic phase block. Why is this suddenly hardcoded?

Reply:
This is part of the Chevron class, not of DynamicPhase. We might want change the Chevron class in the future to accept an op_code instead of taking qbc and qbt individually. Similar as in DynamicPhase, I have now added a note to the class docstring, explaining that the class is currently tailored to CZ gates (with the nomenclature qbc and qbt for control qubit and target qubit), but might be used for other gates by experts.